scipy/stats/tests/test_stats.py

```diff
@@ -7336,6 +7357,20 @@ def test_collapse(self):
         assert_almost_equal(
             stats.wasserstein_distance(u, v, u_weights, v_weights),
             np.average(np.abs(u), weights=u_weights))
+        # test collapse for n dimentional values
+        # Collapsing a n-D distribution to a point distribution at zero
+        # is equivalent to taking the average of the norm of data.
+        u = np.arange(-10, 30, 0.4).reshape((-1, 10))
+        v = np.zeros_like(u)
+        assert_almost_equal(
+            stats.wasserstein_distance(u, v),
+            np.mean(np.linalg.norm(u, axis=1)))
+
+        u_weights = np.arange(len(u))
+        v_weights = u_weights[::-1]
```

Comment:
Is there something special about these weights being reversed relative to the originals?

Reply:
Also no, I have this one updated as well.